components/Main/Experience.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+const featuresTitle = ref<HTMLElement | null>(null);
+
+const items: { title: string }[] = [
+  {
+    title: 'Recepción de pagos por tarjeta o transferencias',
+  },
+  {
+    title: 'Lectura de ubicación por GPS'
+  },
+  {
+    title: 'Chat conversacional'
+  },
+  {
+    title: 'Agendamiento de citas'
+  },
+  {
+    title: 'Almacenamiento en Base de datos'
+  },
+];
+
+onMounted(() => {
+  window.addEventListener('scroll', handleScroll);
+
+  function handleScroll() {
```

Comment:
This function should be outside the on mounted, please take a look on hero component